client/dashboard/src/components/ArticleItem/ArticleItem.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useRef, useEffect } from 'react';
+import styled from 'styled-components';
+
+const FlexColDiv = styled.div`
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const InfoBox = styled.div`
+  background: grey;
+  height: 30px;
+  width: 750px;
+`;
+
+export default function ArticleItem({ article }) {
+  const canvasRef = useRef(null);
+
+  const renderImage = (imageUrl) => {
```

Comment:
You may have multiple items in your list, this means you'll render multiple instances of this component. This function could be declared outside of the component scope and re-used by all the instances in a shared way.